src/monitor.cc

```diff
@@ -172,16 +211,35 @@ void* monitorNodeThread(void *arg) {
     exit(0);
 }
 
-NAUV_WORK_CB(updateLoopTimeStamp) {
+// Invoked when woken by monitr pthread via async_send
+NAUV_WORK_CB(UpdateStatisticsCallback) {
     NodeMonitor::getInstance().setStatistics();
 }
 
+static NAN_GETTER(GetterGCCount) {
+    NodeMonitor& monitor = NodeMonitor::getInstance();
+    GCUsageTracker& tracker = monitor.getGCUsageTracker();
+
+    info.GetReturnValue().Set(Nan::New<Number>(tracker.totalCollections()));
+}
+
+static NAN_GETTER(GetterGCElapsed) {
+    NodeMonitor& monitor = NodeMonitor::getInstance();
+    GCUsageTracker& tracker = monitor.getGCUsageTracker();
+
+    info.GetReturnValue().Set(Nan::New<Number>(tracker.totalElapsedTime() / (1000.0 * 1000.0) ));
```

Comment:
This appears to be returning elapsed seconds, yes? Would elapsed milliseconds be more informative?